app/src/androidTest/java/com/android/shelflife/ui/recipes/AddRecipesTest.kt

```diff
@@ -277,8 +283,7 @@ class AddRecipesTest {
         .onNodeWithTag("inputRecipeInstruction")
         .performTextInput("Add the salmon into the smoker")
 
-    composeTestRule.onNodeWithTag("addButton").performClick()
-
-    verify(navigationActions).goBack()
+    // composeTestRule.onNodeWithTag("addButton").performClick()
```

Comment:
Remember that if you comment certain parts of test/code you must also uncomment them back or remove them entirely.